oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -1074,6 +1078,10 @@ private <T extends Document> T findAndModify(Collection<T> collection,
                 }
             }
 
+            int size = updateOp.toString().length();
```

Comment:
Obtaining the string representation might be expensive, no?

Reply:
Is there any other way to get the Document size?

Reply:
> Obtaining the string representation might be expensive, no?

Is there any other way to get the Document size?